contents/handbook/company/communication.md

```diff
@@ -192,6 +185,45 @@ James (H) and Cory are admins on the Pitch account. Because Pitch charges per se
 1. We use sentence case for titles.
 1. When writing numbers in the thousands to the billions, it's acceptable to abbreviate them (like 10M or 100B - capital letter, no space). If you write out the full number, use commas (like 15,000,000).
 
+## Requests for comment (RFCs)
+
+We use Request For Comments (RFCs) to communicate and gather feedback on a decision.
+
+Here are the steps for an RFC:
+1. Someone identifies a problem and a decision to be made
+2. They create an RFC as a pull request (normally into PostHog/posthog or PostHog/posthog.com unless in the rare case it's confidential where we use PostHog/product-internal) with the title `RFC: ...`. The RFC should include:
+   1. The decision to be made
+   2. Who will make the decision
+      - By default this should be the person who will own the project and do the work. In the rare case that it's a non-reversible decision it might be more appropriate have the team lead be the decision maker.
+   3. When the decision will be made by (normally less than 1 week). You don't need to wait for this date if you've already received input from the key people you need.
+   4. The specific people that you want feedback from (as github assignees)
+      - It can be useful to say which people are required vs which are welcome but optional so others know if they need to respond
+   5. The problem
+   6. The recommendation
+   7. Useful context for the people giving feedback (for product decisions, [this github template](https://github.com/PostHog/product-internal/blob/main/.github/ISSUE_TEMPLATE/request-for-comments.md) is useful)
+3. Share the RFC in the `#github-rfcs` slack channel if it isn't already there
```

Comment:
Either we should have everyone be a member of this channel OR we should post the RFCs in tell-posthog-anything or general

Reply:
I'd suggest just posting them in #general - we don't have that many RFCs and that channel isn't particularly noisy. If it does get too noisy, we could move somewhere else? (vs. creating another Slack channel)

Reply:
Disagree. Right now we already have too many people commenting on RFCs, a lot of times people with only a passing understanding of the problem etc. Better to post the RFC to specific channels that are relevant.

Reply:
Isn't that problem addressed by requesting who you want feedback from as suggested in the template?

Otherwise a pricing RFC (for example) would need to be posted to Growth, Marketing, Website & Docs, CS, and the relevant app team, which seems way noisier.

Reply:
I think there's a difference between people commenting on it vs being aware of the stuff that's going on. There's currently quite a few people are spending quite a lot of time trying to keep up with everything going on in Github e.g. email notifications for any changes across all the repos

I think we can reduce the around this and worry about missing stuff by saying something like: You shouldn't worry about keeping up with everything that's going on especially as the company get bigger. Instead, focus on your team's work and all RFCs will be shared in X channel if you are interested in keeping up to date with stuff across the company.

Reply:
I've added this which I think is a good compromise:

   1. Post in the relevant Slack channel (normally the teams slack channel, or `#general` if it's a bigger cross-team RFC)
   2. Tag the people you need input from in the slack thread or DM them - it's your responsibility to nudge the relevant people for their input
   3. Post in `#github-rfcs` if it isn't already there automatically (so that people interested in RFCs across the company can stay in the loop)